cv32e40s/tb/uvmt/uvmt_cv32e40s_tb_files.flist

```diff
@@ -18,39 +18,39 @@
 ${DV_UVMT_PATH}/uvmt_cv32e40s_dut_wrap.sv
 ${DV_UVMT_PATH}/uvmt_cv32e40s_tb.sv
 
+${DV_UVMT_PATH}/../assertions/uvmt_cv32e40s_fencei_assert.sv
+${DV_UVMT_PATH}/../assertions/uvmt_cv32e40s_pmp_assert.sv
+${DV_UVMT_PATH}/../assertions/uvmt_cv32e40s_pmprvfi_assert.sv
+${DV_UVMT_PATH}/../assertions/uvmt_cv32e40s_umode_assert.sv
```

Comment:
Moved umode and pmp asserts to the assertions dir.

(The other changes are just `sort` in alphabetical order.)